### ISSUE TITLE
EKS numberOfAvailabilityZones isn't supported in Go

### ISSUE THREAD

**Issue by residentorca** [pulumi/pulumi-awsx#1029]

**Comment by kpitzen** [comment]
Hi @residentorca - thank you for opening this issue! I've transferred this issue to our AWSx repo. Looking at the link you've provided, I'm not sure what's missing here. I see this property in arguments on the go SDK:
https://www.pulumi.com/registry/packages/awsx/api-docs/ec2/vpc/#numberofavailabilityzones_go

Can you provide an example program which illustrates the issue? Or describe what's happening that doesn't match your expectations?

Thank you!

**Comment by residentorca** [comment]
You are correct it is supported https://github.com/pulumi/pulumi-awsx/blob/master/sdk/go/awsx/ec2/vpc.go#LL97C21-L97C21

I was looking at the wrong repo https://github.com/pulumi/pulumi-aws/blob/master/sdk/go/aws/ec2/vpc.go

Thank you!.
